rct229/rulesets/ashrae9012019/section5/section5rule1.py

```diff
@@ -0,0 +1,285 @@
+        def __init__(self):
+            super(Section5Rule1.RMDRule, self).__init__(
+                rmrs_used=produce_ruleset_model_instance(
+                    USER=True,
+                    BASELINE_0=True,
+                    BASELINE_90=True,
+                    BASELINE_180=True,
+                    BASELINE_270=True,
+                    PROPOSED=True,
+                ),
+                each_rule=Section5Rule1.RMDRule.BuildingRule(),
+                index_rmr=BASELINE_0,
+                list_path="$.buildings[*]",
+            )
+
+        def create_data(self, context, data):
+            rmd_b0 = context.BASELINE_0
+            rmd_b90 = context.BASELINE_90
+            rmd_b180 = context.BASELINE_180
+            rmd_b270 = context.BASELINE_270
+            rmd_p = context.PROPOSED
+            rmd_u = context.USER
+
+            has_baseline_0 = rmd_b0 is not None
+            has_baseline_90 = rmd_b90 is not None
+            has_baseline_180 = rmd_b180 is not None
+            has_baseline_270 = rmd_b270 is not None
+            has_proposed = rmd_p is not None
+            has_user = rmd_u is not None
+
+            has_basseline_0_output = has_baseline_0 and bool(
+                find_one("$.output.output_instance", rmd_b0)
+            )
+            has_basseline_90_output = has_baseline_90 and bool(
+                find_one("$.output.output_instance", rmd_b90)
+            )
+            has_basseline_180_output = has_baseline_180 and bool(
+                find_one("$.output.output_instance", rmd_b180)
+            )
+            has_basseline_270_output = has_baseline_270 and bool(
+                find_one("$.output.output_instance", rmd_b270)
+            )
+            has_proposed_output = has_proposed and bool(
+                find_one("$.output.output_instance", rmd_p)
+            )
+            has_user_output = has_user and bool(
+                find_one("$.output.output_instance", rmd_u)
+            )
+
+            rmd_list = [rmd_b0, rmd_b90, rmd_b180, rmd_b270, rmd_p, rmd_u]
+            rmd_list_no_user = [rmd_b0, rmd_b90, rmd_b180, rmd_b270, rmd_p]
+
+            # count the rmds that aren't None
+            no_of_rmds = len(compact(rmd_list))
+
+            # filter out rmds that aren't None then count the length
+            no_of_output_instance = len(
+                compact(
+                    [
+                        find_one("$.output.output_instance", rmd)
+                        for rmd in rmd_list_no_user
+                    ]
+                )
+            )
+
+            return {
+                "has_baseline_0": has_baseline_0,
+                "has_baseline_90": has_baseline_90,
+                "has_baseline_180": has_baseline_180,
+                "has_baseline_270": has_baseline_270,
+                "has_proposed": has_proposed,
+                "has_user": has_user,
+                "has_basseline_0_output": has_basseline_0_output,
```

Comment:
select all occurrences of basseline typos and replace with baseline

Reply:
Good catch. Addressed